api/controller/legacy_test.go

```diff
@@ -240,13 +240,15 @@ func (s *legacySuite) TestAPIServerCanShutdownWithOutstandingNext(c *gc.C) {
 	go func() {
 		// Resetting the dummy environment will call Stop on the
 		// embedded API server.
+		start := time.Now()
 		dummy.Reset(c)
+		c.Logf("dummy.Reset() took %v", time.Since(start))
 		close(srvStopped)
 	}()
 
 	select {
 	case <-srvStopped:
-	case <-time.After(testing.LongWait):
+	case <-time.After(time.Minute):
```

Comment:
Let's add a comment and TODO to explain why the timeout is a minute rather than LongWait